README.md

```diff
@@ -1,77 +1,117 @@
 
-## Installation
+## About The Project <a name = "about_the_project"></a>
+Flumes is a tool that allows you to select a folder and detect existing media files automatically. These media files are scanned and all relevant information is stored in your database. In addition, you can always perform a manual scan to make the necessary checks on demand.
 
-## System dependencies
-Besides the dependencies of the package itself, your system needs to have:
-* gstreamer-1.0
-* Gio >=2.62
+**Features**
+* Scan media files in a selected directory
+* Rescan after media directory changes
+* Stores the `GstDiscoveryStream` information and the `GstCaps` structure fields
+* Force rescan manually
 
-## Usage
-You need to provide a configuration on either `$HOME/.flumes` or a system wide configuration at `/etc/flumes`. The database parameters are modeled in the same way as `SQLAlchemy` URL parameters.
+## Getting Started <a name = "getting_started"></a>
+**System requirements** <a name = "system_requirements"></a>
 
-Sample configuration for sqlite3 database driver
-```ini
-[Database]
-uri=sqlite:///flumes.db
-```
+* Operating system: [Ubuntu 20.04](https://releases.ubuntu.com/20.04/) or superior
+  
+* [gstreamer-1.0](https://gstreamer.freedesktop.org/download/)
+  ```
+  gst-launch-1.0 --gst-version
+  ```
+  
+* [Gio >=2.62](https://docs.gtk.org/gio/)
+  ```
+  gio version
+  ```
+  
+* [Python >=3.9, <4.0](https://www.python.org/downloads/)
+* [Docker](https://docs.docker.com/engine/install/ubuntu/) *Only required for installation via Docker. [See here](#docker)*
 
-Link the database to the folder:
-```
-poetry run flumes-discovery -i sqlite:///flumes.db -d <media content directory>  
-```
+**Installation** <a name = "installation"></a>
 
-Alternatively, use Dockerfile for development:
-```
-docker-compose -f docker-compose-dev.yml up
-```
+For a successful and complete installation, you can choose between the Poetry packaging and dependency management tool or just use Docker.
+
+1. If you select via [*Poetry*](https://python-poetry.org/docs/): <a name = "poetry"></a>
 
-## Development
-Install poetry
+* Install poetry
 ```
 curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
 ```
-Install dependencies
+* Install dependencies
 ```
 poetry install
 ```
-Install pre-commit hooks
+* Install pre-commit hooks
 ```
 poetry run pre-commit install
 ```
-
-## New releases
-Upgrade release version number here:
-* __init__.py file
-* tests/test_flumes.py
-* pyproject.toml
-
-Tagging:
+* Link the database to the folder where you want to monitor your media files:
 ```
-git tag -a "<version>" -m "Release <version>"
+poetry run flumes-discovery -i sqlite:///flumes.db -d <media content directory>  
 ```
+2. In case you prefer an alternative to *poetry* for installation, you can use it via [*Docker*](https://www.docker.com/): <a name = "docker"></a>
 ```
-git push origin --tags
+docker-compose -f docker-compose-dev.yml up --build
```

Comment:
Here we need to make it clear to the user that before they can use docker they need to install docker and docker-compose packages